data-type-date-and-time.md

```diff
@@ -151,7 +151,7 @@ TIME[(fsp)]
 
 ### `DATETIME` 类型
 
-`DATETIME` 类型是日期和时间的组合，格式为 `YYYY-MM-DD HH:MM:SS[.fraction]`。支持的范围是 `1000-01-01 00:00:00.000000` 到 `9999-12-31 23:59:59.999999`。`fsp` 参数表示秒精度，取值范围为 0~6，默认值为 0。TiDB 支持字符串或数字转换为 `DATETIME` 类型。
+`DATETIME` 类型是日期和时间的组合，格式为 `YYYY-MM-DD HH:MM:SS[.fraction]`。支持的范围是 `0000-01-01 00:00:00.000000` 到 `9999-12-31 23:59:59.999999`。`fsp` 参数表示秒精度，取值范围为 0~6，默认值为 0。TiDB 支持字符串或数字转换为 `DATETIME` 类型。
```

Comment:
It's wrong.
According to https://dev.mysql.com/doc/refman/8.0/en/datetime.html

> The DATETIME type is used for values that contain both date and time parts. MySQL retrieves and displays DATETIME values in 'YYYY-MM-DD hh:mm:ss' format. The supported range is '1000-01-01 00:00:00' to '9999-12-31 23:59:59'.

Reply:
> It's wrong.
> According to https://dev.mysql.com/doc/refman/8.0/en/datetime.html
> 
> > The DATETIME type is used for values that contain both date and time parts. MySQL retrieves and displays DATETIME values in 'YYYY-MM-DD hh:mm:ss' format. The supported range is '1000-01-01 00:00:00' to '9999-12-31 23:59:59'.

I have get the information from the website. But the fact is users can insert data range of 0000-01-01 00:00:00 to 9999-12-31 23:59:59 into table.

Reply:
> It's wrong.
> According to https://dev.mysql.com/doc/refman/8.0/en/datetime.html
> 
> > The DATETIME type is used for values that contain both date and time parts. MySQL retrieves and displays DATETIME values in 'YYYY-MM-DD hh:mm:ss' format. The supported range is '1000-01-01 00:00:00' to '9999-12-31 23:59:59'.

I don't know how to post a image here. You can try it in TiDB. 
execute these sql:
  create table dtest(dt datetime);
  insert into dtest values('0000-01-01 00:00:00'),('999-01-01 11:22:33');
then
  select * from dtest;
you will find these data can be insert successfully.